dist/common/scripts/scylla_coredump_setup

```diff
@@ -85,8 +85,22 @@ WantedBy=multi-user.target
             run('sysctl -p /usr/lib/sysctl.d/50-coredump.conf', shell=True, check=True)
         else:
             with open('/etc/sysctl.d/99-scylla-coredump.conf', 'w') as f:
-                f.write('kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e"')
+                f.write(f'kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e')
```

Comment:
or, in addition to conflict with `abrt-ccpp.service`, we can just stop the service unit in this script **before**  setting up the pattern file and update the system with it, so that the `abrt-ccpp.service` never rewrites our settings. simpler this way.

Reply:
f-string was typo, dropped.
The fix was removing strange " after %e:
`'kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e"'`